packages/app/src/debug/DebugEmptyStates.cy.tsx

```diff
@@ -0,0 +1,62 @@
+import DebugNotLoggedIn from './DebugNotLoggedIn.vue'
+import DebugNoProject from './DebugNoProject.vue'
+import DebugNoRuns from './DebugNoRuns.vue'
+import DebugLoading from './DebugLoading.vue'
+import DebugError from './DebugError.vue'
+import { useLoginConnectStore } from '@packages/frontend-shared/src/store/login-connect-store'
+
+describe('Debug page empty states', () => {
+  beforeEach(() => {
+    cy.viewport(900, 800)
```

Comment:
Personal preference - I would make this a `describe`-level config just so it doesn't output in the command log on every test. I try to only use `cy.viewport` if I'm trying to demonstrate changes in behavior due to viewport within a given test

```
describe('Debug page empty states', { viewportHeight: 800, viewportWidth: 900 }, () => {
```

Reply:
I ended up just using the default viewport measurements, but I didn't know that we could specify viewport properties this way, will use in the future 👍🏻